### PR TITLE
Merges FT/Det into one role, Investigator.

### PR DESCRIPTION
**Pull request by SueTheCake**
DISCLAIMER: THIS IS A TWO SLOT ROLE. THE NUMBER OF SLOTS HAS NOT CHANGED. TWO!!

By merging detective/CSI into a universal investigator role, it can help alleviate the CBT of them being separate. Reasoning in these mapping changes:

-Sec has bloat. Remove briefing room. Security spawns in the upstairs classroom, which is never used anyway, so by making it the defacto meeting area it can help give the upper level some traffic.

-Investigations and evidence moved to main level. Antags had CBT trying to break in to sabotage or steal things due to their weird placement and only access point being thru 300 security doors or space on the interstitial level. 

-Add a break room in place of old detective's office. You can put your lunchboxes in the fridge, now, wowee. Cool. The training wing gets literally no foot traffic as it is, perhaps now this will provide an RP incentive to be upstairs.

-Regular officers cannot access evidence. The lab has a desk, like the armory or warden's counter. Evidence storage is down to 9 closets, from 12, but I have never used that many, so... it's at least right by the lab.

-Forensics being in the back of security gives it traffic. There was absolutely no reason to be back there post round start anymore.

image

image

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -242,7 +242,7 @@
 /datum/gear/accessory/detbadge
 	display_name = "badge, detective"
```
Comment (Cnaym) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:243]: Should be badge, investigator then. Does not matter for the coat imho, but should be changed for their badge to reflect it ICly

Comment (SueTheCake) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:243]: This is just the name in the loadout, and I do not think it is important. The actual badge simply says that it is 'marks the holder as an investigative agent'.

Comment (carpevenenum) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:243]: I think changing the loadout names would improve user-friendliness for new players, immediately alerting them to the fact that we do not have a separate Detective role.

File: code/game/jobs/jobs.dm [hunk 2]
```diff
@@ -4,16 +4,15 @@ var/const/ENGSEC			=(1<<0)
...
-var/const/FORENSICS			=(1<<12)
+var/const/OFFICER			=(1<<3)
+var/const/CHIEF				=(1<<4)
+var/const/ENGINEER			=(1<<5)
+var/const/ATMOSTECH			=(1<<6)
+var/const/AI				=(1<<7)
+var/const/CYBORG			=(1<<8)
+var/const/INTERN_SEC		=(1<<9)
+var/const/INTERN_ENG		=(1<<10)
+var/const/FORENSICS			=(1<<11)
```
Comment (Arrow768) [inline code/game/jobs/jobs.dm:15]: These flags are used by the job preferences.
So changing them wholesale is not a good idea.
I.e. someone who previously had Atmostech on high would now have AI on high.

Just rename DETECTIVE (1<<3) to Investigator and delete FORENSICS (1<<12)

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (Arrow768) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:243]: Rename that to investigator, as its inconsistent otherwise.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by Arrow768** [comment]
Will be polled on the server.

File: code/game/jobs/jobs.dm (hunk 2 above)
Comment (SueTheCake) [reply code/game/jobs/jobs.dm]: The flag for the role is still FORENSICS, so I moved that to the detective slot and deleted the twelfth slot as you said.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (SueTheCake) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Done.

**Review by Arrow768 — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**